test/e2e/config/azure-dev.yaml

```diff
@@ -51,7 +51,9 @@ providers:
       - sourcePath: "${PWD}/templates/test/cluster-template-prow-ipv6.yaml"
         targetName: "cluster-template-ipv6.yaml"
       - sourcePath: "../data/infrastructure-azure/cluster-template-mhc.yaml"
-        targetName: "cluster-template-mhc.yaml"        
+        targetName: "cluster-template-mhc.yaml"
+      - sourcePath: "${PWD}/templates/test/cluster-template-machine-pool.yaml"
+        targetName: "cluster-template-machine-pool.yaml"
 
```

Comment:
I think you also need to add the new wait times intervals:
 - `wait-machine-pool-nodes`
 - `wait-machine-pool-upgrade`